CONTRIBUTING.md

```diff
@@ -10,9 +10,9 @@ In 2014, you -- the Scala community -- matched the core team at EPFL in number o
 
 We are super happy about this, and are eager to make your experience contributing to Scala productive and satisfying, so that we can keep up this growth. We can't do this alone (nor do we want to)!
 
-This is why we're collecting these notes on how to contribute, and we hope you'll share your experience to improve the process for the next contributor! (Feel free to send a PR for this note, send your thoughts to scala/contributors (Gitter) or contributors.scala-lang.org (Discourse), or tweet about it to @adriaanm.)
+This is why we're collecting these notes on how to contribute, and we hope you'll share your experience to improve the process for the next contributor! (Feel free to send a PR for this note, send your thoughts to scala/contributors (Gitter) or contributors.scala-lang.org (Discourse).)
 
-By the way, the team at Lightbend is: @adriaanm, @lrytz, @retronym, @SethTisue, and @szeiger.
```

Comment:
I put Lukas first because he's lead, but I'm happy to alphabetise instead. it's hard to tell if Adriaan was first because he was lead or because of the alphabet

Reply:
Any order is fine 🤷 , but if you're looking for a rule to follow, the old version was ordered by seniority on the team, with the lead first. So currently that would be: Lukas, Jason, Seth, Dale.

Reply:
It should be ordered by who is your favorite Beatle.

Reply:
did you really get hired in alphabetical order of github handles? 😂

Reply:
> did you really get hired in alphabetical order of github handles?

oh man, that's going to really constrain our future hiring

Reply:
it's too late, Dale already ruined it D:

Reply:
I COME IN LIKE A WREEEECKING BAAALLL.

Reply:
@dwijnand if that's a Miley Cyrus reference, we want video

Reply:
I think the order is OK, it's Lukas, retronym, Seth, and That sbt guy.

Zeke and Zelda sense their time may yet come.